gson/src/main/java/com/google/gson/internal/bind/DateTypeAdapter.java

```diff
@@ -31,10 +31,7 @@
 import java.text.DateFormat;
 import java.text.ParseException;
 import java.text.ParsePosition;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.Locale;
+import java.util.*;
```

Comment:
Your IDE probably did this for you, but please undo it. :-)
This project follows the Google Java Style Guide, notably the bit [excluding wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports).

Reply:
Oh, Idea change it automatically.